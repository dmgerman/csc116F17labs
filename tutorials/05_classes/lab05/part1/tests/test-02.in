13
45
2
26