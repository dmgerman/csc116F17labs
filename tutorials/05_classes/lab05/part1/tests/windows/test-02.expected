Enter initial hour: 
Enter initial minute: 
c.get_hour24() returns 13
c.get_hour12() returns 1
c.get_minute() returns 45
c.get_time_string(true) returns 13:45
c.get_time_string(false) returns 01:45pm
Enter hours to advance: 
Enter minutes to advance: 
Advancing c by 2 hours and 26 minutes.
After advancing, the time is 16:11
